mobile/lib/features/trade/order_submission_status_dialog.dart

```diff
@@ -26,24 +27,47 @@ class OrderSubmissionStatusDialog extends StatelessWidget {
       this.insetPadding = const EdgeInsets.all(50),
       this.navigateToRoute = ""});
 
+  @override
+  State<OrderSubmissionStatusDialog> createState() => _OrderSubmissionStatusDialog();
+}
+
+class _OrderSubmissionStatusDialog extends State<OrderSubmissionStatusDialog> {
+  late final ConfettiController _confettiController;
```

Comment:
❓Why are we defining the `ConfettiController` in the `OrderSubmissionStatusDialog` *and* the `TradeScreen`?

My understanding is that the dialog cannot be closed anymore; so I think it should be enough to define the confetti on the `OrderSubmissionStatusDialog`.

I don't think it's wrong to define it on the underlying `Trade`screen as well, but I'm not sure it's necessary.